backend/src/main/java/com/bakdata/conquery/models/query/resultinfo/printers/CsvResultPrinters.java

```diff
@@ -0,0 +1,185 @@
+package com.bakdata.conquery.models.query.resultinfo.printers;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+import com.bakdata.conquery.internationalization.Results;
+import com.bakdata.conquery.models.common.CDate;
+import com.bakdata.conquery.models.common.daterange.CDateRange;
+import com.bakdata.conquery.models.config.LocaleConfig;
+import com.bakdata.conquery.models.query.C10nCache;
+import com.bakdata.conquery.models.query.PrintSettings;
+import com.google.common.base.Preconditions;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class CsvResultPrinters extends PrinterFactory {
```

Comment:
Würde ich `StringResultPrinter` nennen, da hier nichts CSV spezifisches passieren sollte (escaping sollte von dieser Klasse nicht gehandelt werden) und man das Ergebnis auch in eine HTML Tabelle oder so printen kann